pkg/apis/numaflow/v1alpha1/udf.go

```diff
@@ -190,7 +190,7 @@ type SessionWindow struct {
 // NOTE: Quite powerful, should not be abused; it can cause stalling of pipelines and leaks.
 type AccumulatorWindow struct {
 	// Timeout is the duration of inactivity after which the state of the accumulator is removed.
-	Timeout *metav1.Duration `json:"ttl,omitempty" protobuf:"bytes,1,opt,name=timeout"`
+	Timeout *metav1.Duration `json:"ttl,omitempty" protobuf:"bytes,1,opt,name=ttl"`
```

Comment:
```suggestion
	Timeout *metav1.Duration `json:"ttl,omitempty" protobuf:"bytes,1,opt,name=timeout"`
```

Reply:
it is in aligned meant with session window terminology.

Reply:
Then why don't use `timeout` for all of them?

Reply:
session uses timeout, doesn't it?

Reply:
I meant, in `json`, it uses `ttl`.

Reply:
let me fix this in a follow up?

Reply:
Sounds good, reverted.